amgut/db/0042.sql

```diff
@@ -0,0 +1,129 @@
+CREATE TABLE ag."source" (
+	source_id            uuid DEFAULT ag.uuid_generate_v4() ,
+	source_name          varchar(100)  ,
+	ag_login_id          uuid  ,
+	vioscreen_status     integer  ,
+	CONSTRAINT pk_source UNIQUE ( source_id )
+ );
+
+CREATE INDEX idx_source_1 ON ag."source" ( ag_login_id );
+
+COMMENT ON TABLE ag."source" IS 'A "source" is the human, animal or environment that was swapped for microbial analysis.';
+COMMENT ON COLUMN ag."source".source_id IS 'A unique ID to identify the source.';
+COMMENT ON COLUMN ag."source".source_name IS 'Sources can come with names, e.g. names for the humans that have been swapped.';
+COMMENT ON COLUMN ag."source".ag_login_id IS 'Points to the "user" that owns this source.';
+
+CREATE TABLE ag.source_barcodes_surveys (
```

Comment:
adding NOT NULL as it is done for the other columns? Also in DB schema you can specify all this values. Actually they SHOULD be specified, otherwise the documentation and the code go out-of-sync. DBSchema can generate the patch for the DB structure automatically. If it is wrong, that means that the DBSchema representation is also wrong. Make sure to keep them in sync.